### DISCUSSION TITLE
关于我在ios上配置小火箭的问题

### DISCUSSION THREAD

**Discussion by zypfff** [XTLS/Xray-core#2372] [Q&A]
大家好初来乍到对许多问题都很不解所以来请求大佬的帮助
在我用xui搭建tcp reality的协议后
在小火箭上连接
然后把还原伪装度100的ios下载tk后
发现tk显示没有网络连接  但是视频可以显示出来也可以往下刷就是不播放
然后我查找资料说我小火箭的代理没有开启  但是当我开启小火箭的代理模式后我发现我的节点网络直接用不了
请问大家是不是在协议方面搭建的和tk需要网络配置不符合 还是我手机上的小火箭设置不正确
希望有人能给我一些宝贵的建议谢谢看到的各位！！！！

**Comment by Splaser** [comment]
你开小火箭，然后进你的手机浏览器，看看谷歌、ytb那些网站能不能上。能上就是你tk客户端的问题，上不了就是你小火箭的问题。
